src/components/order/CardSelector.tsx

```diff
@@ -34,16 +34,18 @@ const SelectedImage = styled.img`
 
 const CardSelector = ({ selectedCardId, setValue, theme }: Props) => {
   const handleSelectCard = (cardId: number) => {
-    const isSelected = selectedCardId === cardId;
-    setValue('selectedCardId', isSelected ? null : cardId);
+    const isSelected = selectedCardId === `card${cardId}`;
+    setValue('selectedCardId', isSelected ?null:`card${cardId}`);
     setValue(
       'message',
       isSelected ? '' : orderCard.find((c) => c.id === cardId)?.defaultTextMessage || ''
     );
   };
 
   const selectedCard =
-    selectedCardId !== null ? orderCard.find((c) => c.id === selectedCardId) : null;
+  selectedCardId !== null
+    ? orderCard.find((c) => `card${c.id}` === selectedCardId)
+    : null;
```

Comment:
이건 엄청 중요한 부분은 아닌데요.. 삼항연산자가 굳이 필요없지 않나 싶네요!

```suggestion
  const selectedCard = orderCard.find((c) => `card${c.id}` === selectedCardId)
```

아마 null을 명시적으로 하고 싶으시면 아래와 같은 방법도 좋을 거 같습니다.

```ts
const selectedCard = orderCard.find((c) => `card${c.id}` === selectedCardId) ?? null;
```